### PR TITLE
Docs: Adding SAVE IMAGE tip

### PR DESCRIPTION
**Pull request by adamgordonbell**
Adding tip. 
Had to start new PR and cherry pick @vladaionescu

### CODE REVIEW

File: docs/best-practices/best-practices.md [hunk 1]
```diff
@@ -1167,6 +1168,60 @@ Earthly provides a way to pass-through access to your host's SSH keys to the bui
...
+      FROM ruby:2.7
+      ARG TARGETARCH
+      WORKDIR /site
+      ...
+      RUN apt-get install pandoc  pandocfilters -y
+
+    build:
+      FROM +base
+      # Build blog
+```
```
Comment (vladaionescu) [inline docs/best-practices/best-practices.md:1188]: Nit: this should be unindented.

File: docs/best-practices/best-practices.md [hunk 2]
```diff
@@ -1167,6 +1168,60 @@ Earthly provides a way to pass-through access to your host's SSH keys to the bui
...
+      ...
+      RUN apt-get install pandoc  pandocfilters -y
+
+      # Manually cache the base image by pushing it to a registry
+      SAVE IMAGE –push earthly/blog-base-image:latest
+
+    build:
+      # Use the cached base image for builds  
+      FROM earthly/blog-base-image:latest
+      RUN ...
```
Comment (vladaionescu) [inline docs/best-practices/best-practices.md:1208]: Same with this one

File: docs/best-practices/best-practices.md [hunk 3]
```diff
@@ -1167,6 +1168,60 @@ Earthly provides a way to pass-through access to your host's SSH keys to the bui
...
+```
+
+By doing this, we ensure that the build process is consistently swift, as the base layers do not need to be rebuilt. The trade-off is that the base target needs to be updated manually. For the Earthly website, we address this by running earthly +base once a week via a CI job, and on-demand whenever there are changes to the base image. You can see the full example, including flags for manual rerunning the base image, on GitHub.
```
Comment (vladaionescu) [inline docs/best-practices/best-practices.md:1211]:
```suggestion
By doing this, we ensure that the build process is consistently swift, as the base layers do not need to be rebuilt. The trade-off is that the base target needs to be updated manually. For the Earthly website, we address this by running earthly `+base` once a week via a CI job, and on-demand whenever there are changes to the base image. You can see the full example, including flags for manual rerunning the base image, on GitHub.
```

**Review by vladaionescu — COMMENTED** [review]
Looks good! Do you want to add a small mention within the new `caching-in-earthfiles.md` page?

**Comment by adamgordonbell** [comment]
> Do you want to add a small mention within the new `caching-in-earthfiles.md` page?

Done!

**Review by vladaionescu — APPROVED** [review]

**Pull request merged**